terraform/main.tf

```diff
@@ -0,0 +1,62 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = "~> 1.14"
+}
+
+provider "template" {
+  version = "~> 1.0"
+}
+
+data "aws_ami" "rock" {
+  most_recent = true
+
+  filter {
+    name = "name"
+
+    # Maybe we should rename the AMI to `juxt-rock`?
+    values = ["juxt-arch-*"]
```

Comment:
Maybe I should rename the AMIs to `juxt-rock-*`?

I used Arch because I was thinking about splitting the AMIs in two; one for a more-or-less empty Arch AMI with up-to-date everything and a custom repo for AUR packages, and one for the Clojure stuff we want installed…

Reply:
I've completed the rename in 30be5f7.